src/Components/Menu/ItemCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './ItemCard.css'
+function ItemCard({item}) {
+  return (
+    <div className='ItemCard'>
+        <ul>
+            <li>id: {item.id}</li>
+            <li>name: {item.name}</li>
+            <li>price: ${item.price}</li>
```

Comment:
I think this will display unformatted prices like `$25` (instead of `$25.00`) or `$5.2` (instead of `$5.20`). What method can you use to format the numbers to show two decimal places?